README.md

```diff
@@ -80,11 +81,19 @@ cd CrewLink
 yarn install
 ```
 3. Copy iohook binaries
+
+Mac/Linux:
 ```sh
 mkdir -p node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 cp iohook\iohook.node node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 cp iohook\uiohook.dll node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
 ```
+Windows:
+```sh
+mkdir -p node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
+cp iohook/iohook.node node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
```

Comment:
cp is not a valid command if you don't use `git-bash`. 

Window's native is `copy` instead for command line (or terminal, for those who use VS/VSCode)

Reply:
On top of this, the -p parameter does not apply to the windows version of the command. It'll instead make a folder labelled "-p"

Reply:
you both are very much correct and as you hinted I was indeed using bash in VS Code

Reply:
@sjoerd108 @stevebell117 do yall know what the original commands are actually for? Doesn't mac/linux use `/` for creating directories? I was just trying to be helpful because when using `bash` and Windows there was an extra Visual Studio dependencies and the original commands had strange behavior

Reply:
I'm not quite sure what tools let you use Unix based utilities with windows path separators.

Reply:
I was wondering the same thing